lib/Conch/Controller/DeviceReport.pm

```diff
@@ -214,11 +217,11 @@ sub _record_device_report {
 			) if $dr->{temp};
 
 			$dr->{temp}
-				and $log->info("Recorded environment for Device $device_id");
+				and $log->info("Recorded environment for Device ".$device->id);
```

Comment:
This will not render correctly. use `$device_id`.

Reply:
Addressed